### ISSUE TITLE
Exception column not found `expired` in  `bm_player_warnings`

### ISSUE THREAD

**Issue by oranges13** [ftbastler/BoNeMEAL#103]
the application itself also returns 2 exceptions for the warnings list
![image](https://user-images.githubusercontent.com/22157193/59713612-1c185700-9210-11e9-90ba-51d91be034d5.png)
![image](https://user-images.githubusercontent.com/22157193/59713620-22a6ce80-9210-11e9-9a6e-e5f3daa10fd3.png)

_Originally posted by @Jerlag01 in https://github.com/ftbastler/BoNeMEAL/issues/102#issuecomment-503278194_

**Comment by Jerlag01** [comment]
the tables are created by the latest version of banmanager itself and to my knowledge, nothing has changed to what tables get's created upon first run of the plugin

**Comment by oranges13** [comment]
This would be an issue with your banmanager database, and not an issue with this interface. It's possible that someone renamed or deleted a column in your banmanager table which this program has no control over. 

I would suggest submitting a bug report to them if you have issues with banmanager or its' data structure.

I checked a couple minecraft servers I have access to and `bm_player_warnings` has the `expires` column so it may have been removed or renamed in your installation.
